src/main/java/uk/gov/hmcts/reform/sscs/transformers/SscsCaseTransformer.java

```diff
@@ -302,12 +303,17 @@ private String findSignLanguageType(Map<String, Object> pairs, boolean isSignLan
             for (ScannedRecord record : records) {
                 SscsDocumentDetails details = SscsDocumentDetails.builder()
                     .documentLink(record.getUrl())
-                    .documentDateAdded(record.getScannedDate())
+                    .documentDateAdded(stripTimeFromDocumentDate(record.getScannedDate()))
                     .documentFileName(record.getFileName())
                     .documentType("Other document").build();
                 documentDetails.add(SscsDocument.builder().value(details).build());
             }
         }
         return documentDetails;
     }
+
+    private String stripTimeFromDocumentDate(String documentDate) {
+        DateTimeFormatter dtf = DateTimeFormat.forPattern("yyyy-MM-dd'T'hh:mm:ss.SSS");
```

Comment:
looks like CCD doesn't sends the date in `yyyy-MM-dd'T'hh:mm:ss.SSS'Z'`.

Reply:
@nitinprabhu CCD does send in this format. Test was failing as data was not in correct format

Reply:
I meant Z part after milliseconds